test/mysql-cdc/alter-source.td

```diff
@@ -8,7 +8,7 @@
 # by the Apache License, Version 2.0.
 
 $ set-sql-timeout duration=1s
-$ set-max-tries max-tries=20
+$ set-max-tries max-tries=14
```

Comment:
Why do / can we decrease this value?

Reply:
Turns out I can remove both of these overrides - the tests pass without them